Source/Scene/Batched3DModel3DTileContentProvider.js

```diff
@@ -740,7 +153,7 @@ define([
                 //
                 // We could also make another request for it, but that would make the property set/get
                 // API async, and would double the number of numbers in some cases.
-                that._batchTable = JSON.parse(batchTableString);
+                batchTable._batchTable = JSON.parse(batchTableString);
```

Comment:
Also, it is confusing that `_batchTable` can be too different things depending on the class.  Perhaps the batch table read from the tile should be `_tileBatchTable` or the one used for rendering could be `_runtimeBatchTable` (and rename the class again).